associations.rb

```diff
@@ -0,0 +1,52 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.classroom = self
+  end
+end
+
+class Student
+  attr_accessor :name, :classroom
+
+  def initialize(name)
+    @name = name
+  end
+end
+
+class Rental
+  attr_accessor :book, :person, :date
+
+  def initialize(book, person, date)
+    @book = book
+    @person = person
+    @date = date
+    @book.rentals << self
+    @person.rentals << self
+  end
+end
```

Comment:
- [ ] Great job :+1: n making the rental belongs to a book and also belongs to a person :100: